src/patternfly/components/Chips/chip.scss

```
@@ -0,0 +1,132 @@
+@import "../../patternfly-utilities";
+
+.pf-c-chip {
+  --pf-c-chip--PaddingLeft: var(--pf-global--spacer--sm);
+  --pf-c-chip--MarginTop: calc(#{var(--pf-global--spacer--xs)} / 2);
+  --pf-c-chip--MarginRight: var(--pf-global--spacer--xs);
+  --pf-c-chip--MarginBottom: calc(#{var(--pf-global--spacer--xs)} / 2);
+  --pf-c-chip--BackgroundColor: var(--pf-global--Color--light-100);
+  --pf-c-chip--BorderColor: var(--pf-global--secondary-color--100);
+  --pf-c-chip--BorderWidth: var(--pf-global--BorderWidth--sm);
+  --pf-c-chip--BorderRadius: var(--pf-global--BorderRadius--sm);
+
+  // modified chip
+  --pf-c-chip--m-overflow--BackgroundColor: var(--pf-global--BackgroundColor--light-300);
+  --pf-c-chip--m-overflow--PaddingLeft: 0;
+  --pf-c-chip--m-overflow--BorderColor: transparent;
+  --pf-c-chip--m-overflow--BorderWidth: 0;
+  --pf-c-chip--m-overflow--hover--BorderColor: var(--pf-global--secondary-color--100);
+  --pf-c-chip--m-overflow--hover--BorderWidth: var(--pf-global--BorderWidth--sm);
+
+  // modified chip button
+  --pf-c-chip--m-overflow--button--BorderRadius: var(--pf-global--BorderRadius--sm);
+  --pf-c-chip--m-overflow--button--BorderWidth: 0;
+  --pf-c-chip--m-overflow--button--BorderColor: transparent;
+  --pf-c-chip--m-overflow--button--PaddingRight: var(--pf-global--spacer--sm);
+  --pf-c-chip--m-overflow--button--PaddingLeft: var(--pf-global--spacer--sm);
+  --pf-c-chip--m-overflow--button--active--BorderWidth: var(--pf-global--BorderWidth--sm);
+  --pf-c-chip--m-overflow--button--active--BorderColor: var(--pf-global--secondary-color--100);
+  --pf-c-chip--m-overflow--button--focus--BorderWidth: var(--pf-global--BorderWidth--sm);
+  --pf-c-chip--m-overflow--button--focus--BorderColor: var(--pf-global--secondary-color--100);
+
+  // chip text
+  --pf-c-chip__text--FontSize: var(--pf-global--FontSize--xs);
+  --pf-c-chip__text--Color: var(--pf-global--Color--100);
+  --pf-c-chip__text--MaxWidth: #{pf-size-prem(120px)}; // Hard coded
+
+  // chip button
+  --pf-c-chip__button--PaddingLeft: var(--pf-global--spacer--sm);
+  --pf-c-chip__button--PaddingRight: var(--pf-global--spacer--sm);
+  --pf-c-chip__button--FontSize: var(--pf-global--FontSize--xs);
+
+  position: relative;
+  display: inline-flex;
+  align-items: center;
+  padding-left: var(--pf-c-chip--PaddingLeft);
+  margin-top: var(--pf-c-chip--MarginTop);
+  margin-bottom: var(--pf-c-chip--MarginBottom);
+  color: var(--pf-c-chip--Color);
+  background-color: var(--pf-c-chip--BackgroundColor);
+  border-radius: var(--pf-c-chip--BorderRadius);
+
+  &::before {
+    position: absolute;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+    content: "";
+    border: var(--pf-c-chip--BorderWidth) solid var(--pf-c-chip--BorderColor);
+    border-radius: var(--pf-c-chip--BorderRadius);
+  }
+
+  &:not(:last-child) {
```

should we add an only-child condition too?
```suggestion
  &:not(:last-child),
  &:not(:only-child)
    {
```